packages/protocol/hardhat.config.ts

```diff
@@ -97,31 +97,6 @@ const config: HardhatUserConfig = {
         },
         version: "0.8.9",
     },
-    preprocess: {
-        eachLine: () => ({
```

Comment:
@davidtaikocha If you want to override config values, please add back this section. Ideally, we can just use "LibSharedConfig" as the default production deployment, or you can create a sub-contract of TaikoL1.sol.